pkg/types/agent/agent_config_type.go

```diff
@@ -0,0 +1,34 @@
+package agent
+
+import (
+	aiv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
+	"github.com/openshift/installer/pkg/types/baremetal"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Config or aka AgentConfig is the API for specifying additional
+// configuration for the agent-based installer not covered by
+// install-config.
+type Config struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec Spec `json:"spec,omitempty"`
+	// No status
+}
+
+// Spec contains additional configuration for the agent-based installer
+type Spec struct {
+	// ip address of node0
+	RendezvousIP string `json:"rendezvousIP,omitempty"`
```

Comment:
Can we not use `NodeZeroIP` instead of `RendezvousIP`? Easy to read and understand.

Reply:
https://github.com/openshift/installer/pull/6022#discussion_r900664859